store/src/java/com/zimbra/cs/mime/Mime.java

```diff
@@ -667,9 +667,14 @@ public static Reader getContentAsReader(MimePart textPart, String defaultCharset
     }
 
     public static void recursiveRepairTransferEncoding(MimeMessage mm) throws MessagingException, IOException {
+        String[] xMailerheaderArray = mm.getHeader("X-Mailer");
         for (MPartInfo mpi : listParts(mm, null)) {
             String cte = mpi.mPart.getHeader("Content-Transfer-Encoding", null);
             String ct = getContentType(mpi.mPart);
+            if (xMailerheaderArray != null && xMailerheaderArray[0].contains("Apple Mail")
+                    && mpi.mPart.getDisposition() != null && mpi.mPart.getDisposition().equals(Part.INLINE)) {
```

Comment:
so are we going to convert all inline attachments to normal attachments for mails sent by apple mail client?
what about actual inline images or images inside signatures?

Reply:
@silentsakky, Initially I had made the changes only for PDF files but the issue was occurring for PNG JPG file formats as well hence added for all
As per testing, anything which is inline has to convert to attachment type when viewing in Outlook
or in some or other scenarios it'll cause garbled text in the email
If images in a signature are in JPG or PNG format and if they'll be passed as inline will also cause garbled text in Outlook and they might get changed as attachment in some scenarios as well

Reply:
If there is a single image in Email then it will render as inline and if there are multiple images either in email also in signature, it'll change to attachment